### DISCUSSION TITLE
How to script a migration

### DISCUSSION THREAD

**Discussion by RecRoomRobin** [JetBrains/rider-efcore#122] [Q&A] [answered]
One of the listed features of this plugin is the following:
> Creating SQL scripts from the migration range

I don't see a way to accomplish this. I would have assumed that there would be a 'Script migration' checkbox in the 'Update database' dialog, but there is not.

Is there a way to run something like the following command with this plugin?
```dotnet ef migrations script <from> <to>```

**Comment by seclerp** [comment, marked as answer]
Hi @RecRoomRobin, 

Starting from version **222.0.0** of the plugin there is a dialog for generating scripts for migration range under `Tools > Entity Framework Core >  Generate SQL Script:`

![image](https://user-images.githubusercontent.com/20597871/193905454-2d77fbf6-02d4-4c34-becc-3587cb636481.png)

Note that you should have at least **Rider 2022.2.0**.

More info about recent updates: https://blog.seclerp.me/general/ef-core-inside-rider-large-update/

**Reply by RecRoomRobin** [reply]
Thanks for the pointer, I see the option now with the latest version.
